source/projects/configure-settings.txt

```diff
@@ -42,6 +42,20 @@ Key Handling Options
 Specifies how the ``_id`` field is populated when new documents are
 inserted into a collection. You can choose one of the following options:
 
+- :guilabel:`Autogenerated ObjectId`
+
+  - Your ``_id`` field is populated with an autogenerated random
+    `ObjectId <https://www.mongodb.com/docs/manual/reference/bson-types/#std-label-objectid>`__.
```

Comment:
nit: use the `manual` role instead of hard-coding the link. 
```suggestion
    :manual:`ObjectId </reference/bson-types/#std-label-objectid>`.
```

Reply:
Actually going to convert this to a ref ``:ref:`ObjectId <document-bson-type-object-id>`` since the manual/objects.inv is in the snooty.toml file for this repo.